packages/cli/src/build-single-file.ts

```diff
@@ -175,10 +175,18 @@ export function buildSingleFile({
         // .d.ts.map
         // if not explicitly defined, assumed true with "--dts" parent scope
         if (dtsSourceMap !== false) {
+            const relativeTargetFilePath = relative(
+                dirname(outSrcPath),
+                outputSources ? outSrcPath : filePath
```

Comment:
we might want to fix the source-map to account the namespace reference line we add.
or move that line to the bottom.